CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## dev
 
+- Drop `setuptools` and `wheel` from the shared libraries
```

Comment:
It may be a good idea to describe briefly what this means to end users.

Reply:
Sure, done. Have you already considered adopting `towncrier` for `pipx`? This would also simplify categorizing the changes.